packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -49,8 +49,47 @@ class FrameFlameChart extends CoreElement {
       ..overflow = 'hidden';
 
     enableDragScrolling(this);
+    element.onMouseMove.listen((MouseEvent e) => _handleMouseMove(e));
```

Comment:
you can just write
element.onMouseMove.listen(_handleMouseMove);
same on the following line

Reply:
done.